### PR TITLE
Optimize merging array elements

### PR DESCRIPTION
**Pull request by wgliang**
Optimize merging array elements

### CODE REVIEW

File: server/util.go [hunk 1]
```diff
@@ -209,9 +209,7 @@ func (t *slowLogTxn) Commit() (*clientv3.TxnResponse, error) {
 
 func sliceClone(strs []string) []string {
 	data := make([]string, 0, len(strs))
-	for _, str := range strs {
-		data = append(data, str)
-	}
+	data = append(data, strs...)
```
Comment (disksing) [inline server/util.go]: maybe `copy` is more suitable here.

Comment (wgliang) [reply server/util.go]: Yes, you are right.

Comment (wgliang) [reply server/util.go]: This function maybe can be remove.

Comment (disksing) [reply server/util.go]: Yes. It's unused. Please help to remove it.

**Comment by wgliang** [comment]
It' OK!

**Pull request merged**